### ISSUE TITLE
Feature requests

### ISSUE THREAD

**Issue by JCKodel** [ChangJoo-Park/flutter_foreground_service_plugin#10]
1) Customize the notification (for example, using Big Text Style notification)
![image](https://user-images.githubusercontent.com/379339/88485469-4f1d3600-cf4c-11ea-942c-499c04f0c28f.png)

2) Customize or disable the notification sound (I can't update the notification showing "In N minutes" because every update triggers a sound and would be annoying to the user)

3) Support for minimized notifications (the last one in both example pictures)

4) I don't know how it is called, but be able to change this part of the notification:
![image](https://user-images.githubusercontent.com/379339/88485520-b5a25400-cf4c-11ea-8704-e07f17e4e280.png)

**Comment by FabulousGee** [comment]
For your 4th feature request - this is already implemented! It's called "subtext". Use as follows:
```
void main() {
  runApp(MyApp());
  startForegroundService();

  // if you need to stop foreground service,
  // await FlutterForegroundPlugin.stopForegroundService();
}

void startForegroundService() async {
  await FlutterForegroundPlugin.setServiceMethodInterval(seconds: 5);
  await FlutterForegroundPlugin.setServiceMethod(globalForegroundService);
  await FlutterForegroundPlugin.startForegroundService(
    holdWakeLock: false,
    onStarted: () {
      print("Foreground on Started");
    },
    onStopped: () {
      print("Foreground on Stopped");
    },
    title: "Flutter Foreground Service",
    content: "This is Content",
    iconName: "ic_stat_hot_tub",
    subtext: "JCKodel",
  );
}

void globalForegroundService() {
  debugPrint("current datetime is ${DateTime.now()}");
}
```

Though there is a little bug in the current plugin code - I'll create a pull request shortly and edit my answer here.

**Comment by JCKodel** [comment]
This is the "little" bug?

![image](https://user-images.githubusercontent.com/379339/88962936-886fe180-d27d-11ea-9f56-e82f57bce687.png)

**Comment by FabulousGee** [comment]
Yes, IMHO this is a little bug since it is really easy to find and fix. Got really some other kind of issues with some other plugins :-)

I already created a pull request, so feel free to integrate this in your copy of the plugin.
